students/ZidanLuo/FizzBuzzTest.py

```diff
@@ -0,0 +1,13 @@
+def FizzBuzz():
+	for i in range(1, 101):
```

Comment:
Indentation should be four spaces in Python -- not more, and not <tab> characters.

Make sure your editor is set up to put in four spaces when you hit the <tab> key.